### ISSUE TITLE
browser no response when use igv web

### ISSUE THREAD

**Issue by transorigin** [igvteam/igv-webapp#174]
why I click the igv web page  several times, and then the browser no response

**Comment by helgathorv** [comment]
Do you mean https://igv.org/app? It's working for me.

**Comment by transorigin** [comment]
> Do you mean https://igv.org/app? It's working for me.

Yeah, It's also working for me. But when I use tracks module to view my bam file. It happens when I click the bam area several times.

**Comment by helgathorv** [comment]
We can try to help If you give us precise details of what you did and what you see - including what data you loaded and how, and what you did after loading the data, and at what point IGV stopped behaving as expected and how. Screenshots might help too.

**Comment by transorigin** [comment]
> We can try to help If you give us precise details of what you did and what you see - including what data you loaded and how, and what you did after loading the data, and at what point IGV stopped behaving as expected and how. Screenshots might help too.

thanks, I will provide them as soon as possible

**Comment by transorigin** [comment]
> > We can try to help If you give us precise details of what you did and what you see - including what data you loaded and how, and what you did after loading the data, and at what point IGV stopped behaving as expected and how. Screenshots might help too.
> 
> thanks, I will provide them as soon as possible

hello，

**Comment by transorigin** [comment]
> We can try to help If you give us precise details of what you did and what you see - including what data you loaded and how, and what you did after loading the data, and at what point IGV stopped behaving as expected and how. Screenshots might help too.

hello helgathorv, the detail is when I click this area several times.
![image](https://user-images.githubusercontent.com/32061756/94499983-9ebffc80-0230-11eb-827f-0434c2f33aad.png)
then, the browser begin no response 
![image](https://user-images.githubusercontent.com/32061756/94500118-07a77480-0231-11eb-87c4-d24f8d8d529e.png)

**Comment by helgathorv** [comment]
Hmm. I don't know why it's not responding - I can't reproduce it with any of the data I've tried loading. 

But first, it will be helpful to know what are you hoping to accomplish by clicking in this area multiple times. Double-clicking would normally zoom in, but you're zoomed in to the max (40 bp). Or are you single clicking to bring up the popup info? 

To further debug this, we can try a couple of different things:
If you are familiar with the web browser developer tools, there might be some clues if you open up the console.
Or if your data is available to me via URL, you can share your session with me and I can take look. Click Share, then copy the tinyurl you see in the dialog that pops up and send it to me at igvteam@broadinstitute.org

**Comment by transorigin** [comment]
> Hmm. I don't know why it's not responding - I can't reproduce it with any of the data I've tried loading.
> 
> But first, it will be helpful to know what are you hoping to accomplish by clicking in this area multiple times. Double-clicking would normally zoom in, but you're zoomed in to the max (40 bp). Or are you single clicking to bring up the popup info?
> 
> To further debug this, we can try a couple of different things:
> If you are familiar with the web browser developer tools, there might be some clues if you open up the console.
> Or if your data is available to me via URL, you can share your session with me and I can take look. Click Share, then copy the tinyurl you see in the dialog that pops up and send it to me at [igvteam@broadinstitute.org](mailto:igvteam@broadinstitute.org)

I probably know what caused this problem. You can try click the base then the browser will no response. I think it has nothing to do with the data.

**Comment by transorigin** [comment]
> > Hmm. I don't know why it's not responding - I can't reproduce it with any of the data I've tried loading.
> > But first, it will be helpful to know what are you hoping to accomplish by clicking in this area multiple times. Double-clicking would normally zoom in, but you're zoomed in to the max (40 bp). Or are you single clicking to bring up the popup info?
> > To further debug this, we can try a couple of different things:
> > If you are familiar with the web browser developer tools, there might be some clues if you open up the console.
> > Or if your data is available to me via URL, you can share your session with me and I can take look. Click Share, then copy the tinyurl you see in the dialog that pops up and send it to me at [igvteam@broadinstitute.org](mailto:igvteam@broadinstitute.org)
> 
> I probably know what caused this problem. You can try click the base then the browser will no response. I think it has nothing to do with the data.

such as

**Comment by transorigin** [comment]
![image](https://user-images.githubusercontent.com/32061756/94638062-b7511500-030b-11eb-9961-ce9d7fb2b1ac.png)

**Comment by transorigin** [comment]
> > Hmm. I don't know why it's not responding - I can't reproduce it with any of the data I've tried loading.
> > But first, it will be helpful to know what are you hoping to accomplish by clicking in this area multiple times. Double-clicking would normally zoom in, but you're zoomed in to the max (40 bp). Or are you single clicking to bring up the popup info?
> > To further debug this, we can try a couple of different things:
> > If you are familiar with the web browser developer tools, there might be some clues if you open up the console.
> > Or if your data is available to me via URL, you can share your session with me and I can take look. Click Share, then copy the tinyurl you see in the dialog that pops up and send it to me at [igvteam@broadinstitute.org](mailto:igvteam@broadinstitute.org)
> 
> I probably know what caused this problem. You can try click the base then the browser will no response. I think it has nothing to do with the data. Because I test a lot of data I have.

**Comment by helgathorv** [comment]
I am not able to reproduce the error with any of the data I've tried loading. If you can reproduce it with one of the hosted tracks, or with public data that you can point me to, then I can take a look.

**Comment by transorigin** [comment]
OK，I will share my data by the email which you provided. You can have a try.

**Comment by transorigin** [comment]
Hello, helgathorv. Have you received my data and tested it based on this data.

**Comment by helgathorv** [comment]
@transorigin I apologize, there was a typo in the email address I gave you - it should be igv-team@broadinstitute.org (note the '-'). The other email address does not exist, so I did not receive your email - in fact you should have received a failed delivery notice.

Note that when you send to the corrected address, you will receive an auto-reply about the preferred way to contact the IGV team. Please just ignore that message, but when you see it, you will know that your email was delivered.

**Comment by transorigin** [comment]
> @transorigin I apologize, there was a typo in the email address I gave you - it should be [igv-team@broadinstitute.org](mailto:igv-team@broadinstitute.org) (note the '-'). The other email address does not exist, so I did not receive your email - in fact you should have received a failed delivery notice.
> 
> Note that when you send to the corrected address, you will receive an auto-reply about the preferred way to contact the IGV team. Please just ignore that message, but when you see it, you will know that your email was delivered.

OK, I have resent the data to you. Please check.

**Comment by jrobinso** [comment]
We received the test data and can reproduce the issue.   It is specific to this data, or data like this,  and I think is related to array values in the alignment tags.   The fix will require an igv update, we will update this issue when that happens.

**Comment by transorigin** [comment]
> We received the test data and can reproduce the issue. It is specific to this data, or data like this, and I think is related to array values in the alignment tags. The fix will require an igv update, we will update this issue when that happens.

If this problem is fixed please let me know, thanks.
